src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/navigate-modal/NavigateModal.scss

```diff
@@ -0,0 +1,80 @@
+@import 'src/styles/common';
```

Comment:
Weren't you planning to add a grey background to the navigation modal in this PR?

Reply:
Also, check distances from the left (i.e. from the toolstrip).

Here's what I am seeing in the review deployment:

![image](https://user-images.githubusercontent.com/6834224/208410018-cbaacc7f-8316-4e42-8d12-05a7f6ced5c6.png)

whereas here is what I am seeing in the XD:

![image](https://user-images.githubusercontent.com/6834224/208410103-44070af0-8779-457c-8603-2526e4cb2866.png)

Reply:
Fixed the background colour and created a new ticket to check the paddings: https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1821

✅

Reply:
Check XD for the behaviour of inputs:

https://user-images.githubusercontent.com/6834224/208934407-f71be796-223c-4015-ab9b-9b4f6a05b3e4.mp4

Reply:
Fixed that

✅